.github/workflows/ci.yml

```diff
@@ -87,7 +153,7 @@ jobs:
         uses: actions/github-script@v6
         id: code-coverage-driver
         with:
-          script: 'return "${{ matrix.calculate-code-coverage }}" == "yes" ? "pcov" : "none"'
+          script: 'return "${{ matrix.code-coverage }}" == "yes" ? "pcov" : "none"'
```

Comment:
nothing to act on, but...
i got confused when for some elements you added a verb (`phpdoc-to-type-rules` vs `run-...`), and for some you dropped (`calculate-code-coverage` vs `code-coverage`)

Reply:
Yeah, there were interim changes (like `fixer-mode` vs `run-fixer`) and it made more sense then to use param-like `code-coverage`, but now all the flags are in imperative style, so I'll restore `calculate-` prefix here 🙂.